controller/replica_controller.go

```diff
@@ -366,6 +373,22 @@ func (rc *ReplicaController) CreatePodSpec(obj interface{}) (*v1.Pod, error) {
 		},
 	}
 
+	if r.Spec.BaseImage != "" {
+		pod.Spec.Containers[0].Image = r.Spec.BaseImage
+		pod.Spec.Containers[0].VolumeMounts = append(pod.Spec.Containers[0].VolumeMounts, v1.VolumeMount{
+			Name:      "engine-binaries",
+			MountPath: types.LocalBinaryDirectoryInContainer,
```

Comment:
So `/usr/local/bin` should always be empty inside base image? I think it's safer if we copied the binaries there.

Reply:
The base image is from scratch, so it will always be empty. Recall we don't have `bash` so the copy will need to be added to longhorn replica Go code. What should that look like? Feature flag?

Reply:
If we only allow `scratch` based image (in Rancher VM doc), it should be fine. I just worried that people may found other use for that image. But let's keep it this way then.

Reply:
Yes, I worry that as well. Perhaps we can use a special dir and modify PATH if it becomes a problem.

Reply:
Yeah, let's worry about it later then.